eventdisplay/CMakeLists.txt

```diff
@@ -54,23 +54,28 @@ target_include_directories(${target} PUBLIC
   $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
 )
 
-target_link_libraries(${target} PUBLIC
-  FairRoot::Tools
+target_link_libraries(${target}
+  PUBLIC
   FairRoot::Base # FairRootManager, FairRunAna, FairTSBufferFunctional, FairTimeStamp, FairEventManager
 
+  FairLogger::FairLogger
+
```

Comment:
Hmm, I don't see any `#include.*[lL]ogger` in `eventdisplay/*.h`?

Reply:
If I do

`find . -name "*.h" -exec grep -H include {} \; | cut -f2 -d' ' | sort | uniq
`
I see 

`<fairlogger/Logger.h>`

`find . -name "*.h" -exec grep -H fairlogger {} \;`

gives 

```
./datasource/FairDataSourceI.h:#include <fairlogger/Logger.h>
./datasource/FairTimebasedDataHandlerT.h:#include <fairlogger/Logger.h>
```

Si in my opinion the dependency is correct.

Reply:
Okay, you're right. I missed that for whatever reason.

Marking resolved.